### DISCUSSION TITLE
V3: Undefined array key "modelClass" when using legacy_model_binding

### DISCUSSION THREAD

**Discussion by renky** [livewire/livewire#5841] [7. Bugs (archived)]
After upgrading my application in general everything works now.
But since we made use of model-binding, I switched legacy_model_binding to true.

Finally there are also Components that doesn't have a direct model binding. And in all components like that it throws the exception
Here is a link to an example exception: https://flareapp.io/share/A7EoogBm#debug

I also analyzed content of $valueOrTuple in HandleComponents.php-Class, Method hydrate, Line 211, where this missing value comes from - in my case I see the following content:
```
$valueOrTuple = array:2 [
  0 => []
  1 => array:1 [
    "s" => "elcl"
  ]
]
```
so it is true, it doesn't contain modelClass key

**Comment by joshhanley** [comment]
@renky thanks! That has been fixed and will be in the next release beta.3
